### PR TITLE
Use multi method to get multipe keys at the same time.

### PR DESCRIPTION
**Pull request by spacedmonkey**
Trac ticket: https://core.trac.wordpress.org/ticket/54864
---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by spacedmonkey** [comment]
Correctly implementing this function and priming multiple in a single request, speeds up the unit tests. From 5.28 minutes to 4 minutes.

File: tests/phpunit/includes/object-cache.php [hunk 1]
```diff
@@ -482,6 +487,7 @@ function wp_cache_get_multi_by_key( $server_key, $keys, $groups = '', &$cas_toke
  */
 function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 	global $wp_object_cache;
+	$wp_object_cache->getMulti( $keys, $group );
```
Comment (SergeyBiryukov) [inline tests/phpunit/includes/object-cache.php:492]:
```suggestion

	// Prime multiple keys in a single Memcached call.
	$wp_object_cache->getMulti( $keys, $group );

```

Comment (SergeyBiryukov) [reply tests/phpunit/includes/object-cache.php:492]: It was not clear to me at a glance what this line does, as it does not return the result.

I see now that `getMulti()` retrieves multiple values in a single Memcached request and saves them in the `$cache` property, making subsequent `get()` calls faster.

I think adding a comment to clarify this a bit would be helpful.

**Review by SergeyBiryukov — COMMENTED** [review]
Thanks for the PR!

Left a comment to clarify the `getMulti()` usage. Looks good to me otherwise 👍

**Review by SergeyBiryukov — APPROVED** [review]

**Comment by SergeyBiryukov** [comment]
This was committed in [r54942](https://core.trac.wordpress.org/changeset/54942).

**Pull request closed without merging**